api_spec.yaml

```diff
@@ -576,7 +576,8 @@ paths:
         - OAuth2 Recipe
```

Comment:
flow diagrams:
- remove `if scopes contains 'supertokens.' prefix || scope ==  'openid'` in POST API.
- reserved scopes can be deleted from clients
- flow for remove scopes:
   - start transactions
   - select for update the client
   - delete

Reply:
changes made